src/work-stealing-queue.h

```diff
@@ -35,15 +35,15 @@ static inline ws_array_t *create_ws_array(size_t capacity, int32_t eltsz) JL_NOT
 }
 
 typedef struct {
-    _Atomic(int32_t) top;
-    _Atomic(int32_t) bottom;
+    _Atomic(int64_t) top;
```

Comment:
`Int64_t` is not `size_t`. `size_t` is unsigned and either `uint32_t` on 32bit or `uint64_t` on 64bit.

We should use `size_t` here.